test/export/test_serialize.py

```diff
@@ -416,5 +423,50 @@ def test_model_op_namespace_version_missing_from_deserializer_do_not_raises(self
     TestDeserialize.test_exportdb_supported_case_pytree_flatten
 )
 
+
+@unittest.skipIf(not torchdynamo.is_dynamo_supported(), "dynamo doesn't support")
+class TestSerializeCustomClass(TestCase):
+    def setUp(self):
+        if IS_SANDCASTLE or IS_MACOS or IS_FBCODE:
+            raise unittest.SkipTest("non-portable load_library call used in test")
+        lib_file_path = find_library_location('libtorchbind_test.so')
+        if IS_WINDOWS:
+            lib_file_path = find_library_location('torchbind_test.dll')
+        torch.ops.load_library(str(lib_file_path))
+
+    def test_custom_class(self):
+        custom_obj = torch.classes._TorchScriptTesting._PickleTester([3, 4])
+
+        def f(x):
+            return x + x
+
+        inputs = (torch.zeros(4, 4),)
+        ep = export(f, inputs)
+
+        # Replace one of the values with an instance of our custom class
+        for node in ep.graph.nodes:
+            if node.op == "call_function" and node.target == torch.ops.aten.add.Tensor:
+                arg0, arg1 = node.args
+                node.args = (arg0, custom_obj)
+
+        # Since the exported program is not runnable with our custom obj, we
```

Comment:
Per offline discussion, let's use an op that accepts the custom class object to fully test deserializer logic